keys/hd/hdpath_test.go

```diff
@@ -0,0 +1,73 @@
+package hd
+
+import (
+	"encoding/hex"
+	"fmt"
+	"github.com/tendermint/go-crypto/keys/bip39"
+)
+
+func ExampleStringifyPathParams() {
+	path := NewParams(44, 0, 0, false, 0)
+	fmt.Println(path.String())
```

Comment:
Why does this print instead of checking the result?

Reply:
https://blog.golang.org/examples